src/styles/blocks/_calendar.scss

```
@@ -0,0 +1,38 @@
+.calendar {
+  position: relative;
+  width: $week * ($block-size + $gap) - $gap;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  @each $name, $day in $days {
+    &--start-day--#{$name} &__block:first-child {
+      margin-left: ($day - 1) * $block-size + $gap * ($day - 1);
+    }
+  }
+
+  @for $index from 28 through 31 {
+    &--month-length--#{$index} &__block:nth-child(-n + #{$index}) {
+      @include square($block-size);
+      @extend %text_font;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      box-sizing: border-box;
+      border: 1px solid #000;
```

don't forget to create variables for all colors
```suggestion
      border: 1px solid $c-black;
```